boards/arm/circuitdojo_feather_nrf9160/doc/index.rst

```diff
@@ -0,0 +1,159 @@
+.. _circuitdojo_feather_nrf9160:
+
+nRF9160 Feather
+###############
+
+.. figure:: img/circuitdojo_feather_nrf9160.png
+     :width: 711px
+     :align: center
+     :alt: Circuit Dojo nRF9160 Feather
+
+     nRF9160 Feather (Credit: Circuit Dojo)
+
+Overview
+********
+
+The nRF9160 Feather by Circuit Dojo is a single-board development
+for bringing your LTE-M and NB-IoT applications to life. The circuitdojo_feather_nrf9160
+board configuration leverages the pre-existing support for the Nordic Semiconductor
+nRF9160. Supported nRF9160 peripherals include:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter with EasyDMA)`
+* :abbr:`WDT (Watchdog Timer)`
+* :abbr:`IDAU (Implementation Defined Attribution Unit)`
+
+More information about the board can be found at the
+`nRF9160 Feather Documentation`_.
+
+
+Hardware
+********
+
+.. figure:: img/nrf9160-feather-v31-features.png
+   :width: 711px
+   :align: center
+   :alt: nRF9160 Feather Features
+
+Connections and IOs
+===================
+
+The nRF9160 Feather has everything you know and love about
+the Feather platform. Here are some of the highlights:
+
+LED
+---
+
+* D7 (blue) = P0.03
+
+Push buttons and Switches
+-------------------------
+
+* MODE = P0.12
+* RESET
+
+USB
+---
+
+Contains a USB/UART connection for both debugging and loading new
+code using a UART Enabled MCUBoot.
+
+Standard Battery Connection
+----------------------------
+
+The nRF9160 Feather has a 2 pin battery connector on board. Lithium Polymer batteries >
+300mA required.
+
+Nano SIM Holder
+---------------
+
+The nRF9160 Feather has a built-in nano SIM (4FF) holder located
+on the bottom side.
+
+
+Programming and Debugging
+*************************
+
+circuitdojo_feather_nrf9160 has a Tag Connect TC2030-CTX-NL. It can be used
+by most programmers like:
+
+* J-Link (the nRF53-DK is recommended)
+* CMSIS-DAP based programmers
+
+Check out `Getting Started`_ for more info.
+
+Building an application
+=======================
+
+In most cases you'll want to use the ``_ns`` target with any of the Zephyr
+or Nordic based examples. Some of the examples do not use secure mode,
+so they do not required the ``_ns`` suffix. A great example of this is the
```

Comment:
Now that the non-secure target is named `circuitdojo_feather_nrf9160ns`, this section should be updated.

Reply:
Fixed!